crates/nix_rs/src/flake/outputs.rs

```diff
@@ -135,6 +175,8 @@ impl FlakeOutputsUntagged {
                 "--legacy", // for showing nixpkgs legacyPackages
                 "--allow-import-from-derivation",
                 "--json",
+                "--default-flake-schemas",
+                "github:DeterminateSystems/flake-schemas",
```

Comment:
We should pin this in our top-level `flake.nix` and then used the locked version here in Rust. For reproducibility.

Reply:
You mean parse the `flake.lock` and substitute?

Reply:
No. See how nixci references the devour-flake flake.

https://github.com/srid/nixci/blob/1cab3caea0236eee483e72df312af7ec6ad3c0bd/src/nix/devour_flake.rs#L10-L13

Reply:
done in https://github.com/juspay/omnix/pull/166/commits/9e79b19f291228bc60dae78147499a99cb00a9d1